sql/2020/03_Markup/summary_pages_by_device.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# summary_pages data grouped by device
+
+# live is very cheap
+
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  COUNTIF(TRIM(doctype) <> '') AS freq_doctype,
+  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
```

Comment:
Can this be simplified? `client` is the only field being grouped, so I'm not sure that you need to SUM/OVER. Here and in `pct` fields.
```suggestion
  COUNT(0) AS total,
```